docs/onboarding/8_16/operator/README.md

```diff
@@ -0,0 +1,86 @@
+
+The operator manages:
+
+- [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector)
+- [auto-instrumentation](https://opentelemetry.io/docs/concepts/instrumentation/automatic/) of the workloads using OpenTelemetry instrumentation libraries
+
+## OpenTelemetry Operator
+
+The [kube-stack Helm Chart](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-kube-stack) will be utilized to manage the installation of the Operator's Custom Resource Definitions (CRDs) alongside the configuration of a suite of collectors, which will instrument various components of the Kubernetes environment to enable comprehensive observability and monitoring.
+
+### Daemonset collectors
+
+The OpenTelemetry components deployed within the DaemonSet collectors are responsible for observing specific signals from each node. To ensure complete data collection, these components must be deployed on every node in the cluster. Failing to do so will result in partial and potentially incomplete data.
+
+- Host Metrics: Collects host metrics (hostmetrics receiver) specific to each node.
+- Kubernetes Metrics: Captures metrics related to the Kubernetes infrastructure on each node.
+- Logs: Utilizes a filelog receiver to gather logs from all Pods running on the respective node.
+- OTLP Traces Receiver: Opens an HTTP and a GRPC port on the node to receive OTLP trace data.
+
+### Deployment collector
+
+The OpenTelemetry components deployed within a Deployment collector focus on gathering data at the cluster level rather than at individual nodes. Unlike DaemonSet collectors, which need to be deployed on every node, a Deployment collector operates as a standalone instance.
+
+- Kubernetes Events: Monitors and collects events occurring across the entire Kubernetes cluster.
+- Cluster Metrics: Captures metrics that provide insights into the overall health and performance of the Kubernetes cluster.
+
+### Auto-instrumentation
+
+The Helm Chart is configured to enable zero-code instrumentation using the [Operator's Instrumentation resource](https://github.com/open-telemetry/opentelemetry-operator/?tab=readme-ov-file#opentelemetry-auto-instrumentation-injection) for the following programming languages:
+
+- Go
+- Java
+
+## Configuration
+
+Depending on the deployment model (i.e. self-managed, ESS, serverless), different configuration will be needed.
+
+### On-Prem deployment
+
+All signals including logs, metrics, traces/APM go through the collector directly into Elasticsearch using the ES exporter, a collector's processor pipeline will be used to replace the APM server functionality.
+
+
+1. Create a secret in Kubernetes with the following command.
```

Comment:
first should be added creation of the `opentelemetry-operator-system` namespace

Reply:
good catch, fixed in https://github.com/elastic/opentelemetry/pull/11/commits/1b76a6df280b536a9830ee4d889ea23be88e45be

thanks!